### ISSUE TITLE
Install via pip looks like it does not work (but it does)

### ISSUE THREAD

**Issue by hamogu** [astropy/montage-wrapper#23]
I just tried to install via pip and it looked as if that failed.
Only later did I notice the small note "Successfully installed montage-wrapper" at the bottom.
This might be an issue related to astropy-helpers?

```
/data/guenther/astrosoft> pip install montage-wrapper
Downloading/unpacking montage-wrapper
  Downloading montage-wrapper-0.9.8.tar.gz (208kB): 208kB downloaded
  Running setup.py (path:/tmp/pip_build_guenther/montage-wrapper/setup.py) egg_info for package montage-wrapper
    git submodule command failed unexpectedly:
    fatal: Not a git repository (or any of the parent directories): .git


Requirement already satisfied (use --upgrade to upgrade): astropy in /data/guenther/anaconda/lib/python2.7/site-packages/astropy-1.0.dev10083-py2.7-linux-x86_64.egg (from montage-wrapper)
Requirement already satisfied (use --upgrade to upgrade): numpy>=1.5.1 in /data/guenther/anaconda/lib/python2.7/site-packages (from astropy->montage-wrapper)
Installing collected packages: montage-wrapper
  Running setup.py install for montage-wrapper
    git submodule command failed unexpectedly:
    fatal: Not a git repository (or any of the parent directories): .git


Successfully installed montage-wrapper
Cleaning up...
/data/guenther/astrosoft> 
```

**Comment by Stargrazer82301** [comment]
I get this issue, but it ends in a fatal failure:

```
chris@sputnik:~$ pip install montage-wrapper
Collecting montage-wrapper
  Using cached montage-wrapper-0.9.8.tar.gz
    Complete output from command python setup.py egg_info:
    git submodule command failed unexpectedly:
    fatal: Not a git repository (or any of the parent directories): .git
    
    /opt/anaconda3/lib/python3.5/site-packages/setuptools-35.0.1-py3.5.egg/pkg_resources/__init__.py:183: RuntimeWarning: You have iterated over the result of pkg_resources.parse_version. This is a legacy behavior which is inconsistent with the new version class introduced in setuptools 8.0. In most cases, conversion to a tuple is unnecessary. For comparison of versions, sort the Version instances directly. If you have another use case requiring the tuple, please file a bug with the setuptools project describing that need.
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-k09uds06/montage-wrapper/setup.py", line 25, in <module>
        conf = config.ConfigParser()
    AttributeError: module 'distutils.config' has no attribute 'ConfigParser'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-k09uds06/montage-wrapper/

```

**Comment by ysBach** [comment]
I am now getting exactly the same error....

**Comment by astrofrog** [comment]
@hamogu @ysBach @Stargrazer82301 - apologies for the long delay, I've now released 0.9.9 which fixes compatibility with Python 3.6
